src/components/Home.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { Button } from 'react-bootstrap';
 
 const Home = () => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="text-center">
+      <section>
```

Comment:
Ah sorry I wasn't very clear with my previous comment. I meant to say I don't think we need any sections here — the reason being that `main` can be considered a kind of `section` — so we can remove this `section`

Reply:
Oh, sorry, I got you now.